content/git/concepts/rm/rm.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: 'rm'
+Description: 'The git rm command is used to remove files or directories from a Git repository. When you delete a file using `git rm', Git records the removal, making it easier to track changes in your project's history.
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'Version Control'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+## Deleting files
+
+To remove a file from Git, you have to remove it from your tracked files (from the staging area) and then commit. The git rm command accomplishes this task. It also removes the file from your working directory so that it's no longer an untracked file.
+
+### Deleting a file from the Working Directory
+
+If you delete a files from you working directory, it will appear int he "Changes not staged for commit" section of the `git status` output.
```

Comment:
```suggestion
If  a file is deleted from the working directory, it will appear in the "Changes not staged for commit" section of the `git status` output.
```